### ISSUE TITLE
Error occured while ripping

### ISSUE THREAD

**Issue by shockabukuu** [lauhayden/musedash-ripper#2]
Using v1.3

Log:

Starting rip...
game_dir: C:\Program Files (x86)\Steam\steamapps\common\Muse Dash
output_dir: C:\Users\user\Desktop\home\tools\muse_dash_soundtrack
album_dirs: True
save_covers: False
save_songs_csv: False
Parsing game config...
569 songs in 76 albums found.
Exporting songs...
Exception in rip thread
concurrent.futures.process._RemoteTraceback: 
"""
Traceback (most recent call last):
  File "concurrent\futures\process.py", line 261, in _process_worker
  File "musedash_ripper\core.py", line 356, in export_song
  File "musedash_ripper\core.py", line 268, in extract_music
  File "musedash_ripper\core.py", line 135, in find_with_prefix
FileNotFoundError: Could not find unique bundle file with prefix 'music_assets_disco_night_music_'
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "musedash_ripper\gui.py", line 170, in rip
  File "musedash_ripper\core.py", line 433, in rip
  File "musedash_ripper\core.py", line 181, in parallel_execute
  File "musedash_ripper\core.py", line 172, in parallel_execute
  File "concurrent\futures\_base.py", line 449, in result
  File "concurrent\futures\_base.py", line 401, in __get_result
FileNotFoundError: Could not find unique bundle file with prefix 'music_assets_disco_night_music_'

PS - is there a way to extract the covers in a non-circular format? cheers

**Comment by nob0d7y** [comment]
I think I'm having a similar issue. I've tried renaming all the bundle file names to match the `'music_assets_[title]_music'` prefix that I think the program is looking for, but I still get the same error and log below.

Maybe the filenaming for bundle files has changed with updates to Muse Dash? Bundle files now look like `'music_[title]_assets_all.bundle'`, and there exists only one bundle file that matches that original prefix (that being 'music_assets_hatsune_tenchi_kaibyaku_shinwa_long_music_946ce89f667396e65849fa5476472de4.bundle').

I do believe however, that the csv file is generating correctly. The csv file and its parent directory are the only things that generate when I tick all the options in the program.

```
Starting rip...
game_dir: C:\Program Files (x86)\Steam\steamapps\common\Muse Dash
output_dir: C:\20240703-musedash-music\muse_dash_soundtrack
album_dirs: True
save_covers: True
save_songs_csv: True
Parsing game config...
Saving songs.csv...
569 songs in 76 albums found.
Exporting songs...
Exception in rip thread
concurrent.futures.process._RemoteTraceback: 
"""
Traceback (most recent call last):
  File "concurrent\futures\process.py", line 261, in _process_worker
  File "musedash_ripper\core.py", line 356, in export_song
  File "musedash_ripper\core.py", line 268, in extract_music
  File "musedash_ripper\core.py", line 135, in find_with_prefix
FileNotFoundError: Could not find unique bundle file with prefix 'music_assets_frost_land_music_'
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "musedash_ripper\gui.py", line 170, in rip
  File "musedash_ripper\core.py", line 433, in rip
  File "musedash_ripper\core.py", line 181, in parallel_execute
  File "musedash_ripper\core.py", line 172, in parallel_execute
  File "concurrent\futures\_base.py", line 449, in result
  File "concurrent\futures\_base.py", line 401, in __get_result
FileNotFoundError: Could not find unique bundle file with prefix 'music_assets_frost_land_music_'
```

**Comment by lauhayden** [comment]
Hmm. Likely the bundle naming has changed again. I'll take a look later, but I don't currently have access to my Windows gaming machine so it might be a few days before I have a fix for this one.

> PS - is there a way to extract the covers in a non-circular format? cheers

No, unfortunately it's stored in a circular format in the bundles, I don't think the game actually has the non-circular version in its assets. The files on the wiki aren't named consistently enough to be of use. :(

**Comment by nob0d7y** [comment]
I managed to get it working by poking around at the code. Though, I wasn't able to figure out how to properly deal with 'chaos_glitch' and 'fm_17314_sugar_radio'... https://github.com/nob0d7y/musedash-ripper/commit/412fddaf36c40f200f3c2b680208078a67069159
![image](https://github.com/lauhayden/musedash-ripper/assets/161763443/2f8ff562-b03f-429b-808d-b1c853f8e862)
Everything does extract properly as far as I can tell though!

**Comment by lauhayden** [comment]
Released [v1.3.1](https://github.com/lauhayden/musedash-ripper/releases/tag/v1.3.1) which fixes the above issues.
